Jenkinsfile

```diff
@@ -4,14 +4,22 @@ void CreateCluster(String CLUSTER_PREFIX) {
     withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-key-file', variable: 'CLIENT_SECRET_FILE')]) {
         sh """
             NODES_NUM=3
-            if [ ${CLUSTER_PREFIX} == 'backups' ]; then
+            if [ ${CLUSTER_PREFIX} == 'backups' || ${CLUSTER_NAME} == 'scheduled-backup']; then
                 NODES_NUM=4
```

Comment:
I am sure that  3 nodes is ok now. the test was improved and now we can try to use cluster with 3 nodes